docs/generating-code-from-schema.md

```diff
@@ -225,6 +225,16 @@ String query = new GraphQLQueryRequest(
         new MoviesProjectionRoot<>().movieId()).serialize();
 ```
 
+Kotlin Projects: 
+Kotlin does not support the Java diamond operator (<>) for inferring type arguments. The recommended workaround is to pass in [Nothing] (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-nothing.html) as both arguments.
```

Comment:
I don't think you should call this out as a workaround. Just call out the usage as is.